src/core/preferences.cpp

```diff
@@ -348,6 +348,16 @@ void Preferences::setPreventFromSuspend(bool b)
     setValue("Preferences/General/PreventFromSuspend", b);
 }
 
+bool Preferences::openDestinationFolderOnBalloonClick() const
```

Comment:
qBt may not show balloons on Linux (it uses DBus notifications there) → You have to add this setting conditionally.